ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/PackageCommitter.scala

```diff
@@ -342,6 +342,7 @@ final private[kvutils] class PackageCommitter(
     Executors.newSingleThreadExecutor { (runnable: Runnable) =>
       val t = new Thread(runnable)
```

Comment:
You can simply [use the constructor](https://docs.oracle.com/javase/8/docs/api/java/lang/Thread.html#Thread-java.lang.Runnable-java.lang.String-) for this:
```suggestion
      val t = new Thread(runnable, "package-preload-executor")
```

Reply:
Fixed in 7384e15092046d667e0635194de6110af2a0fa5a.